students/uss_tomasz/lesson_03_functions/snakify_lesson_8_problems.py

```diff
@@ -0,0 +1,64 @@
+"""
+https://snakify.org/lessons/functions/problems/
+"""
+
+
+# Problem The length of the segment
+def distance(x1, y1, x2, y2):
+    return ((x2-x1) ** 2 + (y2-y1) ** 2) ** 0.5
+
+
+first_point_x = int(input("First point, x coordinate = "))
+first_point_y = int(input("First point, y coordinate = "))
+second_point_x = int(input("Second point, x coordinate = "))
+second_point_y = int(input("Second point, y coordinate = "))
+print("Distance between ({0},{1}) and ({2},{3}) equals {4}".format(first_point_x,
+                                                                   first_point_y,
+                                                                   second_point_x,
+                                                                   second_point_y,
+                                                                   distance(first_point_x,
+                                                                            first_point_y,
+                                                                            second_point_x,
+                                                                            second_point_y)))
```

Comment:
**optional hint**
code should rather be concentrated on the left side of the screen to make it easy to follow,
you could extract the args to the list
```
format_args = [
    first_point_x,
    first_point_y,
    second_point_x,
    second_point_y,
    distance(first_point_x,
             first_point_y,
             second_point_x,
             second_point_y)
]

print("Distance between ({0},{1}) and ({2},{3}) equals {4}".format(*format_args)
```
store message
```
message = "Distance between ({},{}) and ({},{}) equals {}"
print(message.format(....crappy args..))
```

or try to use string interpolation https://www.programiz.com/python-programming/string-interpolation maybe it'd look better